src/components/Modes/Week/Week.tsx

```diff
@@ -0,0 +1,81 @@
+import React, { SyntheticEvent, useState } from 'react'
+import { AcceptButton } from '../../ModaleWindow/AcceptButton/AcceptButton'
+import "../ModesCommonSettings.css"
+import "./Week.css"
+
+type WeekProps = {
+    setSettingsIsOpen: Function
+}
+
+export const Week: React.FC<WeekProps> = ({setSettingsIsOpen}) => {
+
+    const selectWeek = (e:SyntheticEvent ) => {
```

Comment:
...Event ) **: void** => { ...

Reply:
```suggestion
    const selectWeek = (e:SyntheticEvent ) => {
```
const selectWeek = (e:SyntheticEvent ):void => {

Reply:
```suggestion
    const selectWeek = (e:SyntheticEvent ):void => {
```
Обязательно указывать тип возвращаемого значения? Он же по умолчанию void